src/main/java/Duke.java

```diff
@@ -19,59 +23,96 @@ public static void main(String[] args) {
                 "You can also add events to the list by typing 'event' " +
                 "followed by task description + '/from' + starting time of event + '/to' ending time of event.\n" +
                 "Type 'list' to view the current todo list.\n" +
-                "Type 'mark' and task event index to mark the task as done and type 'unmark'and index to undo the task.\n" +
+                "Type 'mark' and task event index to mark the task as done " +
+                "and type 'unmark' and index to undo the task.\n" +
                 line;
```

Comment:
I like what you have done as the description of what the bot can do is very clear!